arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -884,6 +884,14 @@
 				<key>descriptionText</key>
 				<string>This sample demonstrates how to query a feature table to get aggregated statistics back for a specific field.</string>
 			</dict>
+			<dict>
+				<key>displayName</key>
+				<string>Viewshed (GeoElement)</string>
+				<key>storyboardName</key>
+				<string>ViewshedGeoElement</string>
+				<key>descriptionText</key>
+				<string>Demonstrates how to construct a viewshed so that it updates automatically relative to a moving GeoElement.</string>
```

Comment:
I think this needs a dependency key for the two files, and they need to be [tagged as on-demand resources](https://devtopia.esri.com/runtime/cocoa-common-sdk/wiki/Create-a-new-sample-for-samples-repo#including-the-sample-into-the-plist).